react/form/src/custom-elements/ConferenceFormElement.js

```diff
@@ -32,42 +32,41 @@ const ATTRIBUTES = {
   id: 'id',
   hidden: 'hidden',
   locale: 'locale',
-  disableDefaultEventHandler: 'disable-default-event-handler', // custom element attribute names MUST be written in kebab-case
+  overrideEventHandler: 'override-event-handler', // custom element attribute names MUST be written in kebab-case
 };
 
 class ConferenceFormElement extends HTMLElement {
-  jss;
-
-  mountPoint;
-
-  keycloak = getKeycloakInstance();
-
-  unsubscribeFromWidgetEvents;
-
-  unsubscribeFromKeycloakEvent;
-
-  onCreate = createWidgetEventPublisher(OUTPUT_EVENT_TYPES.create);
-
-  onUpdate = createWidgetEventPublisher(OUTPUT_EVENT_TYPES.update);
-
-  onErrorCreate = createWidgetEventPublisher(OUTPUT_EVENT_TYPES.errorCreate);
-
-  onErrorUpdate = createWidgetEventPublisher(OUTPUT_EVENT_TYPES.errorUpdate);
-
-  muiTheme;
+  constructor() {
+    super();
+    this.muiTheme = null;
+    this.jss = null;
+    this.mountPoint = null;
+    this.keycloak = getKeycloakInstance();
+    this.unsubscribeFromDefaultWidgetEvents = null;
+    this.unsubscribeFromKeycloakEvent = null;
+    this.onCreate = createWidgetEventPublisher(OUTPUT_EVENT_TYPES.create);
+    this.onCancelEditing = createWidgetEventPublisher(OUTPUT_EVENT_TYPES.cancelEditing);
+    this.onUpdate = createWidgetEventPublisher(OUTPUT_EVENT_TYPES.update);
+    this.onErrorCreate = createWidgetEventPublisher(OUTPUT_EVENT_TYPES.errorCreate);
+    this.onErrorUpdate = createWidgetEventPublisher(OUTPUT_EVENT_TYPES.errorUpdate);
+  }
 
   static get observedAttributes() {
     return Object.values(ATTRIBUTES);
   }
 
+  isAttributeTruthy(attribute) {
+    const val = this.getAttribute(attribute);
+    return val !== undefined && val !== null && val !== 'false';
```

Comment:
just to make sure, empty string will be truthy as well as boolean false (not sure if getAttribute can actually return that)

Reply:
e.g. for hidden attribute the idea was having something like <custom-element hidden> already working. If you have an idea to make it better that's fine of course!

Reply:
Well, I found a better implementation that uses `hasAttribute` method.

Moreover, the "hidden" feature is no more implemented through attribute (so it doesn't rely anymore on `isAttributeTruthy`) because it uses an existing standard, the `hidden` property: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidden